### ISSUE TITLE
Using the DateTime haxelib in a Json typedef crashes cppia

### ISSUE THREAD

**Issue by Justinfront** [nmehost/acadnme#2]
Unknown function : __hxcpp_date_no
Error linking : Unknown global

I can comment round code but thought I should mention the issue, I am unsure what effect this would have on using current time in a cppia app maybe normally none and it is DateTime that needs fixing not yet looked into it.

**Comment by hughsando** [comment]
Yes, I need to either directly import Date into the Cppia host, or the  __hxcpp_date_no symbol.  Thanks for reporting.
